sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class TranslationOperationTests : DocumentTranslationLiveTestBase
```

Comment:
Consider adding `Live` to the name of the class so we know at first glance these are live.
i.e `TranslationOperationLiveTests`